### PR TITLE
Block reports fixes

### PR DESCRIPTION
**Pull request by rsanheim**
**Story card:** [ch132](https://app.clubhouse.io/simpledotorg/story/132/block-level-reports)

Fixing up various things that were discovered for region reports.

### CODE REVIEW

File: app/models/facility_group.rb [hunk 1]
```diff
@@ -83,13 +83,11 @@ def discardable?
   end
 
   def dashboard_analytics(period:, prev_periods:, include_current_period: true)
-    query = DistrictAnalyticsQuery.new(self, period, prev_periods, include_current_period: include_current_period)
-    query.call
+    DistrictAnalyticsQuery.new(self, period, prev_periods, include_current_period: include_current_period).call
```
Comment (rsanheim) [inline app/models/facility_group.rb:86]: Soon these can all be delegated `region` 👍 .

File: app/models/facility.rb [hunk 2]
```diff
@@ -153,13 +153,11 @@ def recent_blood_pressures
   end
 
   def cohort_analytics(period:, prev_periods:)
-    query = CohortAnalyticsQuery.new(self, period: period, prev_periods: prev_periods)
-    query.call
+    CohortAnalyticsQuery.new(self, period: period, prev_periods: prev_periods).call
```
Comment (rsanheim) [inline app/models/facility.rb:156]: Can be delegated to region 🔜 .

File: app/models/facility_district.rb [hunk 3]
```diff
@@ -20,19 +20,11 @@ def organization
...
 
   def dashboard_analytics(period:, prev_periods:, include_current_period: true)
-    query = DistrictAnalyticsQuery.new(
-      self,
-      period,
-      prev_periods,
-      include_current_period: include_current_period
-    )
-    query.call
+    DistrictAnalyticsQuery.new(self, period, prev_periods, include_current_period: include_current_period).call
```
Comment (rsanheim) [inline app/models/facility_district.rb:23]: These will go away one prod data is cleaned up.

File: app/models/region.rb [hunk 4]
```diff
@@ -63,20 +71,12 @@ def facilities
...
 
-  def log_payload
-    attrs = attributes.slice("name", "slug", "path")
-    attrs["id"] = id.presence
-    attrs["region_type"] = region_type
-    attrs["errors"] = errors.full_messages.join(",") if errors.any?
-    attrs.symbolize_keys
+  def dashboard_analytics(period:, prev_periods:, include_current_period: true)
+    DistrictAnalyticsQuery.new(self, period, prev_periods, include_current_period: include_current_period).call
   end
```
Comment (rsanheim) [inline app/models/region.rb:80]: These two analytics methods are needed for other tabs on the region reports.

**Review by rsanheim — COMMENTED** [review]
Some notes for reviewers.

**Comment by prabhanshuguptagit** [comment]
Does this address [this issues](https://sentry.io/organizations/resolve-to-save-lives/issues/2080173902/?environment=sandbox&project=1217715) as well? Putting it here in case it was missed.

**Review by claudiovallejo — APPROVED** [review]

**Pull request merged**